x-pack/test/alerting_api_integration/spaces_only/tests/alerting/group4/bulk_edit.ts

```diff
@@ -463,7 +461,7 @@ export default function createUpdateTests({ getService }: FtrProviderContext) {
       });
     });
 
-    it('should bulk update API key with apiKey operation', async () => {
+    it('should not bulk update API key with apiKey operation', async () => {
```

Comment:
should we throw an error here? I am wondering...

Reply:
Not sure why we should, we just don't want to update the revision field